src/wallet/db.cpp

```diff
@@ -463,15 +463,6 @@ void CDBEnv::CloseDb(const std::string& strFile)
     }
 }
 
-bool CDBEnv::RemoveDb(const std::string& strFile)
```

Comment:
@laanwj I'd vote for removing it until someone finds a need for it. Motivation: such a dangerous operation should be tested :-)